src/components/TodosList.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { PreparedTodo } from '../interfaces';
+import { Todo } from './Todo';
+
+
+interface Props {
+  todos: PreparedTodo[];
+  onSortTitle: () => void;
+  onSortName: () => void;
+  onSortStatus: () => void;
+}
+
+export const TodosList: FC<Props> = (props) => {
+  const {
+    todos,
+    onSortName,
+    onSortStatus,
+    onSortTitle,
+  } = props;
+
+  return (
+    <>
+      <button type="button" onClick={onSortTitle}>Sort by title</button>
+      <button type="button" onClick={onSortName}>Sort by name</button>
+      <button type="button" onClick={onSortStatus}>Sort by status</button>
```

Comment:
Создавать три отдельных метода не лучший вариант.

Создаешь метод
```
const handleSortTodos = (field: string) => {
  switch (field) {
    case 'name':
      break;
    default:
  }
};
```
```suggestion
      <button type="button" onClick={() => handleSortTodos('name')}>
        Sort by title
      </button>
      ..other buttons
```